java-frontend/src/main/java/org/sonar/java/se/FlowComputation.java

```diff
@@ -101,6 +101,8 @@ private FlowComputation(Set<SymbolicValue> symbolicValues, Predicate<Constraint>
     return result;
   }
 
+  // FIXME It is assumed that this will always return set with at least one element, which could be empty, because result is consumed in other places and messages are
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Take the required action to fix the issue indicated by this comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1134)